Model/Api/Client.php

```diff
@@ -837,7 +837,7 @@ public function refund(\Magento\Sales\Model\Order\Payment $payment, $amount)
             'id' => $transactionId,
             'amount' => $amount * 100,
             // @todo fix incorrect items amount
```

Comment:
Is this todo fixed?

I'm a bit worried that this PR might break certain setups. If any item amounts are wrong, refunds for Walley will fail.

Reply:
@andersem , we had changes in the prepareSalesItems method related to amount calculation after this line was commented.